vscode-lean4/src/leanclient.ts

```diff
@@ -470,7 +471,7 @@ export class LeanClient implements Disposable {
     }
 
     // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
-    sendNotification(method: string, params: any): void {
+    sendNotification(method: string, params: any): Promise<void> | undefined{
```

Comment:
this is probably due to the vscode version change, please revert this since Gabriel wants this to be done in a separate PR.